ui/src/icons/candlestick.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="18" height="16" viewBox="0 0 18 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
do you use this?

Reply:
Not exactly; I copied the svg code into the `CandlestickIcon` component directly, but decided to keep the original file in case we needed to reference it. we can remove it if you don't think we'll need to...

Reply:
Id say you can remove it